### PR TITLE
python quickstart & better quickstart logging

### PR DESCRIPTION
**Pull request by CakeCrusher**
- Ignore dependency package management (initialized when downloading openai for js quickstart)
- python quickstart (copy of js quickstart)
- clearer logging for both quickstarts

### CODE REVIEW

File: examples/quickstart.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ async function getCurrentWeather(location) {
 async function createAssistant() {
     const assistant = await openai.beta.assistants.create({
         instructions: "You are a weather bot. Use the provided functions to answer questions.",
-        model: "mistralai/mixtral-8x7b-instruct",
+        model: "gpt-3.5-turbo-1106",
```
Comment (louis030195) [inline examples/quickstart.js:16]: why did you change to openai though?

Comment (CakeCrusher) [reply examples/quickstart.js]: no reason just personal covinience, forgot to revert fixed.

**Comment by louis030195** [comment]
@CakeCrusher thanks a lot for the contribution! Highly appreciated 🙏
